### ISSUE TITLE
Process suggestion

### ISSUE THREAD

**Issue by iamwillbar** [package-url/purl-spec#3]
The single PR with comments is becoming hard to follow. I'd suggest merging the current PR as a version 0.1 of the spec. We can then use issues and pull requests so we can have one artifact per issue rather than trying to track it all in a single artifact.

**Comment by pombredanne** [comment]
@iamwillbar you are completely right!
Let me incorporate the latest feedback and merge in as a 0.1 alriught

**Comment by pombredanne** [comment]
All done. Closing.
